run/README.md

```diff
@@ -0,0 +1,90 @@
+# Google Cloud Run Java Samples
+
+[![Open in Cloud Shell][shell_img]][shell_link]
+
+[shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
+[shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=blog/README.md
+
+This directory contains samples for [Google Cloud Run](https://cloud.run). [Cloud Run][run_docs] runs stateless [containers](https://cloud.google.com/containers/) on a fully managed environment or in your own GKE cluster.
+
+## Samples
+
+|           Sample                |        Description       |     Deploy    |
+| ------------------------------- | ------------------------ | ------------- |
+|[Hello World][helloworld]&nbsp;&#10149; | Quickstart | [<img src="https://storage.googleapis.com/cloudrun/button.svg" alt="Run on Google Cloud" height="30">][run_button_helloworld] |
+|[Cloud Pub/Sub][pubsub] | Handling Pub/Sub push messages | [<img src="https://storage.googleapis.com/cloudrun/button.svg" alt="Run on Google Cloud" height="30">][run_button_pubsub] |
+
+For more Cloud Run samples beyond Java, see the main list in the [Cloud Run Samples repository](https://github.com/GoogleCloudPlatform/cloud-run-samples).
+
+## Setup
+
+1. [Set up for Cloud Run development](https://cloud.google.com/run/docs/setup)
+
+2. Clone this repository:
+
+    ```
+    git clone https://github.com/GoogleCloudPlatform/java-docs-samples.git
+    ```
+
+    Note: Some samples in the list above are hosted in other repositories. They are noted with the symbol "&#10149;".
+
+
+## How to run a sample locally
+
+1. [Install docker locally](https://docs.docker.com/install/)
+
+2. [Build the sample container](https://cloud.google.com/run/docs/building/containers#building_locally_and_pushing_using_docker):
+
+    ```
+    export SAMPLE=$sample
```

Comment:
What's going on here?  What are you trying to say?

Reply:
updated.